config/_config.yml

```diff
@@ -250,3 +250,40 @@ fontawesome:
   - version: 3.2.1
     stylesheet: 'https://maxcdn.bootstrapcdn.com/font-awesome/3.2.1/css/font-awesome.min.css'
     stylesheet_sri: sha256-MmuZTsWcczT1IhH71aqQmja5jRcXy3mL/NOvjUy9tso= sha512-IJ+BZHGlT4K43sqBGUzJ90pcxfkREDVZPZxeexRigVL8rzdw/gyJIflDahMdNzBww4k0WxpyaWpC2PLQUWmMUQ==
+showcase:
```

Comment:
Feel free to ignore this, just a suggestion, but the key's feel a bit verbose here. I would use `name`, `img`, `lib`.